src/app/products/products-routing.module.ts

```diff
@@ -203,6 +204,14 @@ const routes: Routes = [
                   resolve: {
                     shareProductAndTemplate: ShareProductAndTemplateResolver
                   }
+                },
+                {
+                  path: 'createdividend',
```

Comment:
Routing is incorrect, see routing of other create forms, also there is dividends list page #804  before initiate dividend form,